mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -97,7 +97,6 @@ final class GBTClassifier @Since("1.4.0") (
 
   @Since("1.4.0")
   override def setSeed(value: Long): this.type = {
```

Comment:
If I remove this method I get compile errors in "JavaGBTClassifierSuite". I believe this has something to do with the fact that `setSeed` in class `DecisionTreeParams` returns `this.type`, which is a mixed in trait for the GBTClassifier class. I also checked RandomForestClassifier/Regressor and they have a `override def setSeed(value: Long): this.type = super.setSeed(value)` setter in the final class.